extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -158,6 +162,10 @@ require([
 		if (exitstitial) {
 			exitstitial.init();
 		}
+
+		if (uapContext.getUapId()) {
```

Comment:
Is it working? `BOTTOM_LEADERBOARD` should be initalized no matter if UAP is loaded. We have rubicon configured for that slot.